saleor/graphql/webhook/subscription_query.py

```diff
@@ -0,0 +1,140 @@
+from enum import Enum, Flag
+from typing import Dict, List, Optional, Union
+
+from django.core.exceptions import ValidationError
+from graphene.utils.str_converters import to_snake_case
+from graphql import get_default_backend, validate
+from graphql.error import GraphQLSyntaxError
+from graphql.language.ast import (
+    Document,
+    Field,
+    FragmentDefinition,
+    FragmentSpread,
+    InlineFragment,
+    OperationDefinition,
+)
+
+
+class IsFragment(Flag):
+    TRUE = True
+    FALSE = False
+
+
+class SubscriptionQueryError(Enum):
+    MISSING_SUBSCRIPTION = "Subscription operation can't be found."
+    MISSING_EVENT_FIELD = "Event field can't be found."
+    MISSING_EVENTS = "Can't find a single event."
```

Comment:
Maybe instead of returning generic errors in API, we should return these specific errors to give the consumer more precise info about what is wrong. We would have to add these to schema with some descriptions then.